src/overlays/actors/ovl_En_Peehat/z_en_peehat.c

```diff
@@ -92,85 +99,987 @@ static ColliderJntSphInit D_80AD2790 = {
         COLSHAPE_JNTSPH,
     },
     1,
-    D_80AD276C,
+    sJntSphElemInit,
+};
+
+static ColliderQuadInit sQuadInit = {
+    {
+        COLTYPE_METAL,
+        AT_ON | AT_TYPE_ENEMY,
+        AC_ON | AC_HARD | AC_TYPE_PLAYER,
+        OC1_NONE,
+        OC2_NONE,
+        COLSHAPE_QUAD,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0xFFCFFFFF, 0x00, 0x10 },
+        { 0xFFCFFFFF, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_NONE,
+    },
+    { { { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f } } },
+};
+
+typedef enum {
+    DMG_EFF_ATTACK = 0,
+    DMG_EFF_LIGHT_ICE_ARROW = 6,
+    DMG_EFF_FIRE = 12,
+    DMG_EFF_HOOKSHOT = 13,
+    DMG_EFF_BOOMERANG = 14,
+    DMG_EFF_NUT = 15
+} DamageEffect;
```

Comment:
can you put some offset comments
and also youll want to make the typename of the enum more specific to peahats